src/main/scala/api/commands/blastn.scala

```diff
@@ -0,0 +1,85 @@
+package ohnosequences.blast.api
+
+import ohnosequences.cosas._, types._, records._, klists._
+
+// type blastn = blastn.type
+case object blastn extends AnyBlastCommand {
+
+  type Arguments = arguments.type
+  case object arguments extends RecordType(db :×: query :×: out :×: |[AnyBlastOption])
+
+  type Options = options.type
+  case object options extends RecordType(
+    num_threads     :×:
+    task            :×:
+    evalue          :×:
+    max_target_seqs :×:
+    strand          :×:
+    word_size       :×:
+    show_gis        :×:
+    ungapped        :×:
+    |[AnyBlastOption]
+  )
+
+  type ArgumentsVals =
```

Comment:
I did this before, it is incredibly ugly. If you think that the benefits of having a concrete type outweigh the unfathomable ugliness, so be it.

Anyway, we could generate it implicitly instead of copy-pasting, right?